fe_repo/src/pages/ChatBox.tsx

```diff
@@ -63,54 +63,53 @@ export const ChatBox = ({ onSendMessage, onAnalyze }: IChatBoxProps) => {
       }
       {/* icon row */
       }
-      <div className="flex justify-start gap-2 mb-2">
-        <div
-          style={{ width: '64px', display: 'flex', justifyContent: 'center', cursor: 'pointer' }}
-          onClick={() => {
-            // Upload files
-            const handleFiles = (event: Event) => {
-              const files = (event.target as HTMLInputElement)?.files;
-              if (!files || files.length === 0) return;
-              const file = files[0];
-              if (file) {
-                // upload if file is valid
-                uploadFile(file);
-              }
-            }
-
-            const input = document.createElement("input");
-            input.type = 'file';
-            input.accept = '.pdf';
-            input.onchange = handleFiles;
-            input.click();
-          }}
-        >
-          <UploadIcon />
+        <div className="flex justify-start gap-2 mb-2">
```

Comment:
The indent has been changed to 4, but it should be 2 by our lint rule. Could you revert it back? @Stanford997 @andyasdd1

Reply:
Done~